### ISSUE TITLE
Sidenav sticky style causing overlap with content on production | (Bugs in production cannot be replicated in development)

### ISSUE THREAD

**Issue by andersonmmi** [makerdao/community-portal#423]
**Describe the bug**
Sidenav renders in front of content in mobile view when accessing the production site. Additionally, other UI components overflow the borders of the screen.

**To Reproduce**
1. Go to https://community-portal-staging.makerfoundation.com/en/learn/ from a mobile device or viewer.
2. Refresh or swipe down to refresh the page.
3. See the bug.

4. Go to http://localhost:8000/en/learn/ from a mobile device or viewer.
5. Refresh or swipe down to refresh the page.
6. Don't see the bug.

**Expected behavior**
Sidenav should not be visible and UI components should not overflow the view screen.

**Screenshots**
_on production, at the staging site_
![Screen Shot 2020-09-02 at 6 26 37 PM](https://user-images.githubusercontent.com/16314437/92047103-dcbd2400-ed49-11ea-9e5c-9e1a570f4643.png)

_on r2d, at localhost:8000_
![Screen Shot 2020-09-02 at 6 27 01 PM](https://user-images.githubusercontent.com/16314437/92047130-ec3c6d00-ed49-11ea-972e-12412fbe8b00.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome (inspector)

**Smartphone (please complete the following information):**
 - Device: iPhone5, Galaxy note 9
 - OS: iOS, Android
 - Browser: Crome, Brave

**Additional context**
Some bugs that exist in production cannot be replicated in development. This makes it difficult to submit the appropriate bugfix. This may be a side effect of production being 17 commits ahead of r2d.

**Comment by andersonmmi** [comment]
@MaximumCrash this bug first appeared after @twblack88 merged #414 into r2d, then r2d into production. This is the sx for the Sidenav `<Box/>`:
```
{{
        flexGrow: 0,
        position: "relative",
        maxHeight: "1012px",
        height: "calc(100vh)",
        pt: "54px",
        pl: 3,
        pr: 2,
        pb: "90px",
        width: "307px",
        overflow: 'auto',
        borderRight: "1px solid",
        borderColor: "muted"
}}
```

**Comment by twblack88** [comment]
I'm sorry if it broke it y'all! But this gives us an opportunity to find out why `prod` is 17 commits ahead of `r2d` or if that even matters. 🙇

**Comment by MaximumCrash** [comment]
For the sake of this issue's history, this issue is about the sidenav showing up in production but not showing up in the dev server. This is because production builds are static, there's no server rendering for handling the sidenav's re-rendering. During static builds the sticky will override the styles since emotion is a little played back. 

TLDR: Server side rendering renders twice, once on the server one on the client side. If the client is behind the server it will update based on what's provided by the server (iirc). On client side, there's only 1 render provider, so there's nothing to handle corrections.

**Comment by MaximumCrash** [comment]
Fixed by #427
